index.js

```diff
@@ -184,7 +185,13 @@ function resolveWebpackPath(partial, filename, directory, webpackConfig) {
 
   try {
     var loadedConfig = require(webpackConfig);
-    var resolver = webpackResolve.create.sync(loadedConfig.resolve || {});
+    var resolveConfig = objectAssign({}, loadedConfig.resolve);
+
+    if (resolveConfig.root) {
+      resolveConfig.modules = [].concat(resolveConfig.root);
```

Comment:
I'm not sure, to be honest.  We could always go the current route and add your suggestion if it becomes necessary. It's your call entirely.